myokit/tests/test_formats_importers.py

```diff
@@ -50,6 +50,12 @@ def test_unknown(self):
 
 class SBMLTest(unittest.TestCase):
     """ Test SBML import. """
+    # Test Case I: Hodgkin Huxley
+    i = formats.importer('sbml')
+    HoHu = i.model(os.path.join(
+        DIR_FORMATS,
+        'sbml',
+        'HodgkinHuxley.xml'))
```

Comment:
Also, there's a `setUpClass` method you can use to initialise a unit test, instead of declaring class properties like this. See e.g. https://github.com/MichaelClerx/myokit/blob/master/myokit/tests/test_simulation_cvode.py#L34

Reply:
We can make multiple Test Classes for multiple models later, if you like

Reply:
Ok, sounds good! I used the setUpClass method now.